src/Transport/Receiving/MessageReceiverNotifier.cs

```diff
@@ -253,66 +253,66 @@ async Task ProcessMessage(IMessageReceiverInternal internalReceiver, BrokeredMes
 
                 var context = new BrokeredMessageReceiveContextInternal(message, entity, internalReceiver.Mode);
 
-                    var scope = wrapInScope
-                        ? new TransactionScope(TransactionScopeOption.RequiresNew, new TransactionOptions
-                        {
-                            IsolationLevel = IsolationLevel.Serializable
-                        }, TransactionScopeAsyncFlowOption.Enabled)
-                        : null;
+                var scope = wrapInScope
```

Comment:
Not sure what's going on in the diff here from just looking at it on GitHub. What changed here?

Reply:
Formatting, the code was fixed as part of the formatting rule.